tests/storage/merge_test.py

```diff
@@ -305,14 +305,19 @@ class TestFinalizeMerge:
     def make_env(self, sd_type='block', format='raw', chain_len=2):
         size = MiB
         base_fmt = sc.name2type(format)
+        if sd_type == 'block' and format == 'raw':
+            prealloc = sc.PREALLOCATED_VOL
+        else:
+            prealloc = sc.SPARSE_VOL
```

Comment:
Don't add logic here, the test should configure this properly to test all cases.